<launch>

    <arg name="markerSize"      default="0.05"/>    <!-- in m -->
    <arg name="side"             default="left"/>
    <arg name="ref_frame"       default="base"/>  <!-- leave empty and the pose will be published wrt param parent_name -->


    <node pkg="aruco_ros" type="marker_pub" name="aruco_marker_publisher">
        <remap from="/camera_info" to="/cameras/$(arg side)_hand_camera/camera_info" />
        <remap from="/image" to="/cameras/$(arg side)_hand_camera/image" />
        <param name="image_is_rectified" value="True"/>
        <param name="marker_size"        value="$(arg markerSize)"/>
        <param name="reference_frame"    value="$(arg ref_frame)"/>   <!-- frame in which the marker pose will be refered -->
        <param name="camera_frame"       value="$(arg side)_hand_camera"/>
    </node>

</launch>

